src/SFML/Audio/AudioDevice.cpp

```diff
@@ -108,7 +108,7 @@ AudioDevice::~AudioDevice()
 
 
 ////////////////////////////////////////////////////////////
-bool AudioDevice::isExtensionSupported(const std::string& extension)
+bool AudioDevice::isExtensionSupported(const std::filesystem::path& extension)
```

Comment:
Is this really a path and not just the extension (e.g. .flac)?

Reply:
I used `std::filesystem::path` because `std::filesystem::path::extension` returns a `std::filesystem::path`. Apparently that's how the standard library views extensions.

Reply:
I think you're totally right. Now that I look more deeply, I don't see any references to filesystem extensions. See #2024. Thanks!